docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc

```diff
@@ -0,0 +1,100 @@
+[id="migrating-mgr-from-controller-nodes_{context}"]
+
+= Migrating Ceph Mgr daemons to {Ceph} nodes
+
+The following section describes how to move Ceph Mgr daemons from the
+OpenStack controller nodes to a set of target nodes. Target nodes might be
+pre-existing {Ceph} nodes, or OpenStack Compute nodes if Ceph is deployed by
+{OpenStackPreviousInstaller} with an HCI topology.
+
+.Prerequisites
+
+Configure the target nodes (CephStorage or ComputeHCI) to have both `storage`
+and `storage_mgmt` networks to ensure that you can use both {Ceph} public and
+cluster networks from the same node. This step requires you to interact with
+{OpenStackPreviousInstaller}. From {rhos_prev_long} {rhos_prev_ver} and later
+you do not have to run a stack update.
+
+.Procedure
+
+This procedure assumes that cephadm and the orchestrator are the tools that
+drive the Ceph Mgr migration. As done with the other Ceph daemons (MDS,
+Monitoring and RGW), the procedure uses the Ceph spec to modify the placement
+and reschedule the daemons. Ceph Mgr is run in an active/passive fashion, and
+it's also responsible to provide many modules, including the orchestrator.
```

Comment:
This paragraph should move after the very first paragraph of this procedure:
....or OpenStack Compute nodes if Ceph is deployed by director with an HCI topology.

This procedure assumes that...

.Preqrequisites

Reply:
Makes sense, I'll do in the next iteration.